x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/AuthorizationService.java

```diff
@@ -568,9 +568,12 @@ private ElasticsearchSecurityException denialException(Authentication authentica
         }
         // check for run as
         if (authentication.getUser().isRunAs()) {
+            logger.debug("action [{}] is unauthorized for user [{}] run as [{}]", action, authUser.principal(),
```

Comment:
I don't really see much value in having these log messages when we have an audit log. Can you explain how they help?